tests/foreman/cli/test_hammer.py

```diff
@@ -183,16 +189,81 @@ def test_positive_disable_hammer_defaults(self):
             Defaults.add({'param-name': 'organization_id', 'param-value': default_org['id']})
             # Verify --organization-id is not required to pass if defaults are set
             result = ssh.command('hammer product list')
-            self.assertEqual(result.return_code, 0)
+            assert result.return_code == 0
             # Verify product list fail without using defaults
             result = ssh.command('hammer --no-use-defaults product list')
-            self.assertNotEqual(result.return_code, 0)
-            self.assertFalse(default_product_name in "".join(result.stdout))
+            assert result.return_code != 0
+            assert not (default_product_name in "".join(result.stdout))
```

Comment:
`assert default_product_name not in "".join(result.stdout) ` like it  ? non-blocking